cedar-policy/src/api.rs

```diff
@@ -872,9 +872,12 @@ impl Authorizer {
     ///
     /// The authorizer uses the `stacker` crate to manage stack size and tries to use a sane default.
     /// If the default is not right for you, you can try wrapping the authorizer or individual calls
-    /// to `is_authorized` in `stacker::grow`. Note that `stacker` does not provide support in some
-    /// platforms like `wasm32`. In such cases, the authorizer will assume that the stack size is
-    /// sufficient.
+    /// to `is_authorized` in `stacker::grow`.
+    /// Note that on platforms not supported by `stacker` (e.g., `wasm32`, Android),
```

Comment:
I'd say "Wasm" here like you did in the changelog.
```suggestion
    /// Note that on platforms not supported by `stacker` (e.g., `wasm32`, Android),
```